packages/rest/src/__tests__/acceptance/file-upload/file-upload-with-parser.acceptance.ts

```diff
@@ -109,7 +109,7 @@ class MultipartFormDataBodyParser implements BodyParser {
             value: {
               files: request.files,
               // tslint:disable-next-line:no-any
-              fields: (request as any).fields,
+              fields: request.body,
```

Comment:
@truongkhanhduy95 Please elaborate the proposed change? I think `multer` sets `fields`.